internal/provider/resource_private_key.go

```diff
@@ -71,49 +90,71 @@ func resourcePrivateKey() *schema.Resource {
 			"ecdsa_curve": {
 				Type:        schema.TypeString,
 				Optional:    true,
-				Description: "ECDSA curve to use when generating a key",
+				Description: "Curve to use when generating an ECDSA key",
 				ForceNew:    true,
 				Default:     "P224",
```

Comment:
Probably can't change it now, but P-256 is probably a saner default given the restrictions around SSH protocol not supporting P-224. :-)

Reply:
yeah, discussed this yesterday with @kmoe too - it's very unfortunate, but changing it now will definitely break code that exists out there :(